### ISSUE TITLE
Undocumented Wunderlist delete task-comment endpoint

### ISSUE THREAD

**Issue by duncan** [wunderlist/api#6]
http://stackoverflow.com/questions/32878211/undocumented-wunderlist-delete-task-comment-endpoint#new-answer

There appears to be an endpoint on the Wunderlist API for deleting a task-comment which does not appear in the official docs

Existing task comments can be deleted by sending a DELETE request to it, e.g. :

curl \
    -H "X-Access-Token: $WL_ACCESS_TOKEN" \
    -H "X-Client-ID: $WL_CLIENT_ID" \
    -XDELETE \
    https://a.wunderlist.com/api/v1/task_comments/12345?revision=1
Assuming the data is valid, this returns with HTTP 204.

Is this endpoint supported?

**Comment by vsmart** [comment]
Thank you for collecting these! Is unsupported to us the same as undocumented? Aka, as soon as we document it, it's supported? What else needs to be done?

**Comment by duncan** [comment]
Yep. If we document it, it becomes something we need to support. In the case of this API, I'm pretty sure we didn't want third party clients deleting comments, even with a proper access token. @nathan-ms or @octatone, can you weigh in?
